stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/serializers.py

```diff
@@ -1,17 +1,17 @@
 """Serializers."""
 import abc
 import json
-from datetime import datetime
 from typing import TypedDict
 
 import attr
 import geoalchemy2 as ga
-from stac_pydantic.shared import DATETIME_RFC339
```

Comment:
DATETIME_RFC339 has been deprecated in stac_pydantic. It has several problems including, not parsing fractional seconds or timezones other than Z.